docs/resources/function.md

```diff
@@ -21,18 +21,21 @@ description: |-
 - `name` (String) Specifies the identifier for the function; does not have to be unique for the schema in which the function is created. Don't use the | character.
 - `return_type` (String) The return type of the function
 - `schema` (String) The schema in which to create the function. Don't use the | character.
-- `statement` (String) Specifies the javascript / java / sql code used to create the function.
+- `statement` (String) Specifies the javascript / java / sql / python code used to create the function.
 
 ### Optional
 
 - `arguments` (Block List) List of the arguments for the function (see [below for nested schema](#nestedblock--arguments))
 - `comment` (String) Specifies a comment for the function.
-- `handler` (String) the handler method for Java function.
-- `imports` (List of String) jar files to import for Java function.
+- `handler` (String) the handler method for Java and python function.
+- `imports` (List of String) jar files to import for Java function or for importing python files.
 - `language` (String) The language of the statement
 - `null_input_behavior` (String) Specifies the behavior of the function when called with null inputs.
+- `packages` (List of String) For java the value should be of the form package_name:version_number, where package_name is snowflake_domain:package and for python use it as packages = ('numpy','pandas','xgboost==1.5.0').
```

Comment:
List of package imports to use for Java / Python functions. For Java, package imports should be of the form: package_name:version_number, where package_name is snowflake_domain:package. For Python use it should be:  ('numpy','pandas','xgboost==1.5.0').